pages/index.tsx

```diff
@@ -677,21 +678,13 @@ const Home: NextPage = ({ modelDetails, shortlinkImageParams }: any) => {
             <ArrowBarLeftIcon />
           </Button>
         </FlexRow>
-        {hasValidationError && (
-          <div className="mt-2 text-red-500 font-semibold">
-            Please correct all input errors before continuing
-          </div>
-        )}
-        {hasError && hasError === fixedSeedErrorMsg && (
-          <div className="mt-2 text-amber-400 font-semibold">
-            {fixedSeedErrorMsg}
-          </div>
-        )}
-        {hasError && hasError !== fixedSeedErrorMsg && (
-          <div className="mt-2 text-red-500 font-semibold">
-            Error: {hasError}
-          </div>
-        )}
+
+        <FormErrorMessage
+          hasValidationError={hasValidationError}
+          hasError={hasError}
+          fixedSeedErrorMsg={fixedSeedErrorMsg}
+        />
```

Comment:
(Bear with me as I step through this commit by commit).

I like this idea of creating an specific error component. This particular feedback is more for me, but those props make me think that I should really standardize the error messages the web app is generating. Will add to the to-do list and definitely not a blocker for the PR.

Reply:
Oh, GitHub. Now that I view this in the files changed panel, this comment is attached nowhere close to where I was commenting on. 🙃